mobile/package-lock.json

```diff
@@ -17,6 +17,7 @@
         "react": "18.0.0",
         "react-hook-form": "7.33.1",
         "react-native": "0.69.1",
+        "react-native-bootsplash": "^4.2.3",
```

Comment:
You shouldn't have modified version in package.json manually, in package-lock.json it still isn't fixed. You should install it with -E flag

Reply:
Can't find any information about -E flag. 
![image](https://user-images.githubusercontent.com/82529236/183287341-f65aa988-2f61-4af1-a399-e5dc94a7c677.png)
CanI use only:  "react-native-bootsplash": "4.2.3",  in package-lock.json ?

Reply:
<img width="647" alt="Screenshot 2022-08-07 at 13 05 09" src="https://user-images.githubusercontent.com/64521658/183287654-8b9c4a11-9017-4f99-a1f0-368f3b6e064b.png">

Reply:
https://docs.npmjs.com/cli/v8/commands/npm-install

Reply:
Ohh, sorry, I didn't understand you correctly.